app-text/editorconfig-core-c/editorconfig_core_c-0.12.6.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="EditorConfig core library written in C"
+DESCRIPTION="EditorConfig makes it easy to maintain the correct coding style when switching \
+between different text editors and between different projects. The EditorConfig project maintains \
+a file format and plugins for various text editors which allow this file format to be read and \
+used by those editors. For information on the file format and supported text editors, see the \
+EditorConfig website."
+HOMEPAGE="http://editorconfig.org/"
+COPYRIGHT="2011-2013 EditorConfig Team, including Hong Xu and Trey Hunner
+	2014 Hong Xu"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/editorconfig/files/EditorConfig-C-Core/$portVersion/Source/editorconfig-core-c.tar.gz"
+CHECKSUM_SHA256="8c18dea5faa5105f83466ebee88f1cbc262d8912f546f8b33f547541c821446d"
+SOURCE_DIR="editorconfig-core-c"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	editorconfig_core_c$secondaryArchSuffix = $portVersion
+	cmd:editorconfig$commandSuffix = $portVersion
+	cmd:editorconfig_$portVersion$commandSuffix = $portVersion
+	lib:libeditorconfig$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpcre2_8$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	editorconfig_core_c${secondaryArchSuffix}_devel = $portVersion
+	devel:libeditorconfig$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	editorconfig_core_c$secondaryArchSuffix == $portVersion base
```

Comment:
Should I add `devel:libpcre2_8$secondaryArchSuffix` here?

Reply:
Seeing it depends on it probably yes.